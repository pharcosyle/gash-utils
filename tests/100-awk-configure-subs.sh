awk -f tests/data/awk/subs.awk tests/data/awk/subs.in
