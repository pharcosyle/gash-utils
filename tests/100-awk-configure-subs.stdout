#! meek
BASH=gash #foobar
LIBS=
CC_FOR_BUILD=@CC_FOR_BUILD@

all:
	mescc -o bar bar.c

# end
